tests/test_integration_ftp.py

```
@@ -0,0 +1,476 @@
+            msp_size = msp_file.stat().st_size
+            assert msp_size > 0, f"MSP file is empty: {msp_file}"
+            
+            # Check MSP format
+            with open(msp_file, 'r') as f:
+                content = f.read(500)
+                assert 'Name:' in content or 'NAME:' in content.upper(), \
+                    f"MSP file doesn't contain expected format: {msp_file}"
+        
+        logger.info(f"Successfully created {len(msp_files)} MSP file(s) with 'best' strategy")
+        
+    except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)